components/BottomNavigation/tests/unit/BottomNavigationItemViewCodingTests.m

```diff
@@ -89,4 +94,40 @@ - (void)testMultipleCodingsKeepSubviewsEqual {
   XCTAssertEqual(view.subviews.count, unarchivedView.subviews.count);
 }
 
+- (void)testContentsLayout {
```

Comment:
Please add Given-When-Then comments and break this up into individual test cases.